.github/workflows/verify.yml

```diff
@@ -4,8 +4,8 @@
 name: Latest commit
 
 env:
-  CACHE_VERSION: 6
-  DEFAULT_PYTHON: "3.9"
+  CACHE_VERSION: 7
+  DEFAULT_PYTHON: "3.10"
```

Comment:
I won't recommend that, as this allows for sneaking in 3.10 features, that break 3.9 unexceptedly.

Reply:
Good call, just as much I'm not sure why it's breaking at black on 3.9 references now ... I'll look at it again with a fresh pair of eyes sometime later today.